octokit/users.py

```diff
@@ -0,0 +1,14 @@
+import uritemplate
+
+class Users(object):
+    def get_user(self, inputUser=None, **kwargs):
+        if inputUser:
+            return self.user(user=inputUser)
+        else:
+            return self.current_user
+
+    def get_all_users(self):
+        url = uritemplate.expand(self.user.url, {})
+        if(url[-1] == '/'):
+            url = url[:-1]
+        return self.paginate(url[:-1], {})
```

Comment:
@eduardoramirez Should I have done self.user.paginate()? I tried that, but I needed to process the url with the trailing '/'. I tried to do that in resource but for some reason affects other calls, like getting the client